config/carbonapi.yaml

```diff
@@ -29,29 +29,37 @@ cache:
        - "127.0.0.1:11211"
 # Amount of CPUs to use. 0 - unlimited
 cpus: 0
-#graphiteWeb: "graphiteWeb.example.yaml"
+
 # Timezone, default - local
 tz: ""
-# If 'true', carbonapi will send requests as is, with globs and braces
-# Otherwise for each request it will generate /metrics/find and then /render
-# individual metrics.
-# true --- faster, but will cause carbonzipper to consume much more RAM.
-#
-# For some backends (e.x. graphite-clickhouse) this *MUST* be set to true in order
-# to get reasonable performance
+
+# Deprecated and removed:
+# sendGlobsAsIs: true|false
+# alwaysSendGlobsAsIs: true|false
+# maxBatchSize: int
+# Migration path 
+# alwaysSendGlobsAsIs: true -> resolveGlobs: 0
+# sendGlobsAsIs: false -> resolveGlobs: -1
+# sendGlobsAsIs: true && maxBatchSize: int -> resolveGlobs: int
+
+resolveGlobs: 100
+# = 0 - faster (no 'find in advance', direct render)
+# < 0 - slower (always 'find in advance' and every metric rendered individually)
```

Comment:
Would it make sense to change it from `<0` to simply `1`? Meaning resolveGlobs=1  enforces /metrics/find and separate /render for each metric. This way the behaviour could be clear w/o reading description. While for `<0` behaviour is not self-evident w/o reading docs.

Reply:
I agree with @timtofan here. With your proposal, negative values and 1 will result in the same way. So why bother define a new condition after all?

Reply:
Understood. This is a quite elegant solution!